apps/expo-go/android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.kt

```diff
@@ -230,19 +230,24 @@ class Kernel : KernelInterface() {
               localBundlePath
             )
           )
-          val hostWrapper = ReactNativeHostWrapper(applicationContext, nativeHost)
-          reactHost = ReactHostFactory.createFromReactNativeHost(applicationContext, hostWrapper)
 
-          if (nativeHost.devSupportEnabled) {
+          if (!KernelConfig.FORCE_NO_KERNEL_DEBUG_MODE &&
+            exponentManifest.getKernelManifestAndAssetRequestHeaders().manifest.isDevelopmentMode()
```

Comment:
nit: maybe store a `val manifest = exponentManifest.getKernelManifestAndAssetRequestHeaders().manifest` first because we have many `exponentManifest.getKernelManifestAndAssetRequestHeaders().manifest...` there